.travis.yml

```diff
@@ -1,21 +1,29 @@
 language: go
 
 dist: xenial
-sudo: required
 
 go:
   - 1.12.x
   - 1.13.x
   - 1.14.x
+
 env:
   - GO111MODULE=on
 
-before_install: go get golang.org/x/tools/cmd/goimports
+before_install:
+  - go get golang.org/x/tools/cmd/goimports
+  - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.27.0
 
 script:
   - diff -u <(echo -n) <(gofmt -d -s .)
   - diff -u <(echo -n) <(goimports -d .)
   - go test -v -race ./... -coverprofile=coverage.txt -covermode=atomic
+  - golangci-lint run -v --disable-all
```

Comment:
Maybe we could put the linter before running `go test`?

Reply:
All right. I'll modify it